### PR TITLE
Implement a workaround for issue #590

### PR DESCRIPTION
**Pull request by ctron**
Signed-off-by: Jens Reimann <jreimann@redhat.com>

### CODE REVIEW

File: kura/org.eclipse.kura.camel.cloud.factory/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -19,6 +19,7 @@ Import-Package: org.apache.camel;version="[2.17.2,3.0.0)",
  org.eclipse.kura.cloud.factory;version="[1.1,1.2)",
  org.eclipse.kura.configuration;version="[1.1,1.2)",
  org.eclipse.kura.configuration.metatype;version="[1.0,2.0)",
+ org.eclipse.kura.core.configuration;version="1.0.2",
```
Comment (MMaiero) [inline kura/org.eclipse.kura.camel.cloud.factory/META-INF/MANIFEST.MF]: @ctron why we are not using version range here for backward compatibility reasons?
Thanks

File: kura/org.eclipse.kura.camel.cloud.factory/src/org/eclipse/kura/camel/cloud/factory/internal/CamelManager.java [hunk 2]
```diff
@@ -147,15 +146,16 @@ private static String asString(Object object) {
 
     @Override
     public ComponentConfiguration getConfiguration() throws KuraException {
-        return new ComponentConfiguration() {
+        // FIXME: replace with ComponentConfiguration instance once issue #590 is fixed
+        return new ComponentConfigurationImpl() {
 
             @Override
             public String getPid() {
```
Comment (MMaiero) [inline kura/org.eclipse.kura.camel.cloud.factory/src/org/eclipse/kura/camel/cloud/factory/internal/CamelManager.java]: @ctron Just out of curiosity, why are you not using the constructor, but instead using an anonymous class?
Thanks

Comment (ctron) [inline kura/org.eclipse.kura.camel.cloud.factory/src/org/eclipse/kura/camel/cloud/factory/internal/CamelManager.java]: Just out of laziness. If we fix it up in the next release the change back again just renaming.

File: kura/org.eclipse.kura.camel.cloud.factory/META-INF/MANIFEST.MF (hunk 1 above)
Comment (ctron) [inline kura/org.eclipse.kura.camel.cloud.factory/META-INF/MANIFEST.MF]: I will change this one.

**Pull request merged**